### PR TITLE
Add cr22rc to members list

### PR DESCRIPTION
**Pull request by cr22rc**
Signed-off-by: rickr <cr22rc@users.noreply.github.com>

### CODE REVIEW

**Comment by cr22rc** [comment]
@evankanderson   lgm ?

File: peribolos/knative.yaml [hunk 1]
```diff
@@ -113,6 +113,7 @@ orgs:
     - coryrc
     - cppforlife
     - craigbox
+    - cr22rc
```
Comment (grantr) [inline peribolos/knative.yaml:116]: tiny nit: isn't `cr22rc` ahead of `craigbox` alphabetically?

**Pull request merged**